tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/util/ServerUtil.java

```diff
@@ -487,23 +487,29 @@ public static PortalBundle getPortalBundle(IProject project)  throws CoreExcepti
 
         IStatus status = sdk.validate();
 
-        if ( !status.isOK())
+        if ( !status.isOK() )
         {
             return null;
         }
 
-        Map<String, Object> appServerProperties = sdk.getBuildProperties();
+        final Map<String, Object> appServerProperties = sdk.getBuildProperties(false);
```

Comment:
Lets make a new sdk.getBuildProperties() method that just defaults to getBuildProperties(false)